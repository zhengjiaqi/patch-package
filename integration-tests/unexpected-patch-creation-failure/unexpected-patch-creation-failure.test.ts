import { runIntegrationTest } from "../runIntegrationTest"
runIntegrationTest({
  projectName: "unexpected-patch-creation-failure",
  shouldProduceSnapshots: true,
})
