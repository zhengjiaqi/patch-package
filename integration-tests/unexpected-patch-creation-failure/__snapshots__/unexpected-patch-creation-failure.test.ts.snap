// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test unexpected-patch-creation-failure: patch-package fails to parse a patch it created 1`] = `
"SNAPSHOT: patch-package fails to parse a patch it created

⛔️ ERROR
        
  patch-package was unable to read the patch-file made by git. This should not
  happen.
  
  A diagnostic file was written to
  
    ./patch-package-error.json.gz
  
  Please attach it to a github issue
  
    https://github.com/ds300/patch-package/issues/new?title=New+patch+parse+failed&body=Please+attach+the+diagnostic+file+by+dragging+it+into+here+🙏
  
  Note that this diagnostic file will contain code from the package you were
  attempting to patch.


END SNAPSHOT"
`;
